src/google-classroom-extractor/tests/api/courses/courses-sync.csv

```diff
@@ -0,0 +1,9 @@
+id,name,section,descriptionHeading,description,room,ownerId,creationTime,updateTime,enrollmentCode,courseState,alternateLink,teacherGroupEmail,courseGroupEmail,guardiansEnabled,calendarId
```

Comment:
initial test has sync table with one course with update, two courses that are in both but unchanged, and one course not in the regular table